src/main/java/org/mockbukkit/mockbukkit/inventory/InventoryMock.java

```diff
@@ -681,4 +686,15 @@ public Inventory getSnapshot()
 		return inventory;
 	}
 
+	public @NotNull Component getTitle()
```

Comment:
hasTitle() would be helpful for validation since getTitle is never null.

Reply:
I have divided this into 3 functions:
- getCustomTitle & setCustomTitle: Get/Sets the custom name for that inventory (can be null)
- getTitle: Retrieves the inventory name by deciding when to use the default name or the custom name.